### DISCUSSION TITLE
Intellij / jetbrains licenses for OSS

### DISCUSSION THREAD

**Discussion by pskowronek** [mucommander/mucommander#1099] [General]
@ahadas maybe it is worth to submit the request?

https://www.jetbrains.com/community/opensource/#support

**Comment by ahadas** [comment]
@pskowronek ack, I've applied. IIRC we had such a license for Intellij in the past but it was long time ago (and I personally switched to Eclipse since then, but I'm willing to give Intellij another chance :) )

**Comment by ahadas** [comment]
@pskowronek I shared a license with you by email
